packages/gasket-redux/package.json

```diff
@@ -75,6 +79,10 @@
       "unicorn/filename-case": "error"
     }
   },
+  "eslintIgnore": [
+    "lib",
+    "coverage"
```

Comment:
nit: should we standardize on this?

Reply:
These may be the last couple outliers, so yes I think standardizing on `"eslint ."` while using `eslintIgnore` will be complete.